ydb/docs/en/core/troubleshooting/spilling/total-size-limit-exceeded.md

```diff
@@ -0,0 +1,34 @@
+# Total size limit exceeded
+
+Maximum total size of spilling files exceeded (parameter `max_total_size`). This occurs when the total size of all spilling files reaches the configured limit, preventing new spilling operations.
```

Comment:
Link to the parameter documentation in `reference/configuration/`  from here instead of repeating it in a note at the bottom of this article.